syntax = "proto3";

package test.v1;


service EchoService {
    rpc handler(EchoRequest) returns (EchoResponse) {
    }

    rpc error_handler(EchoRequest) returns (EchoResponse) {
    }

	rpc blocking(Empty) returns (stream EchoResponse) {
	}

	rpc unblock(Empty) returns (Empty) {
	}
}


message EchoRequest {
    string name = 1;
}

message EchoResponse {
    string name = 1;
}

message Empty {
}
